src/cli/run.js

```diff
@@ -122,23 +118,37 @@ console.log(Object.keys(resolved).length, 'resolved queries');
 
 /** Step (4) */
 
+const schemaCache = {};
+const getCachedSchemas = (schemaFilePath: string) => {
+    if (!schemaCache[schemaFilePath]) {
+        schemaCache[schemaFilePath] = getSchemas(
+            makeAbsPath(schemaFilePath, path.dirname(absConfigPath)),
+        );
+    }
+
+    return schemaCache[schemaFilePath];
+};
+
 let validationFailures: number = 0;
 const printedOperations: Array<string> = [];
 
 Object.keys(resolved).forEach((filePathAndLine) => {
     const {document, raw} = resolved[filePathAndLine];
 
-    if (
-        config.crawl.excludes?.some((rx) => new RegExp(rx).test(raw.loc.path))
-    ) {
-        return; // skip
-    }
-
     const hasNonFragments = document.definitions.some(
         ({kind}) => kind !== 'FragmentDefinition',
     );
     const rawSource: string = raw.literals[0];
 
+    const generateConfig = findApplicableConfig(
```

Comment:
nitpick: Looks like we overwrite the general config here, with no option to merge with the root config, so all options have to be explicitly copied and pasted.

This is just a nice-to-have that was in the previous subconfig PR, but can we add the option to do a merge rather than completely override?

Reply:
Yeah I decided to make each generate config completely independent (instead of having a "main + subconfigs"). Given that you can write it as a `.js` file now, you can spread in any shared options in https://github.com/Khan/webapp/blob/gqf-multi/services/static/dev/graphql-flow/config.js. I hope that makes it easy enough, while keeping behavior simple.